frontend/model/contracts/members/removeMemberSideEffect.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+import sbp from '~/shared/sbp.js'
```

Comment:
NOTE: @taoeffect 
I had to move this function out of `contracts/group.js`, otherwise, it would create a circular dependency when imported at `proposals.js` because that file also imports `contracts/group.js`.

Not sure if this is the best place... any suggestion?